### DISCUSSION TITLE
Plugin shortcodes used in posts appear as-is in RSS output (possibly in other places)

### DISCUSSION THREAD

**Discussion by followmyrandomness** [GetPublii/Publii#1803] [Bug Report]
### Operating system

Linux Mint

### Publii version

0.46.2 (build 16966)

### Issue type

Application

### Bug description

Sorry for the confusing explanation; I'll try my best to clarify what I mean. I've noticed this behavior with the Lucide Icons plugin, and it might also occur with other plugins. When I use a shortcode inside a post—such as [li=music size=1.3rem]—the text appears exactly as-is in the RSS feed (i.e., [li=music size=1.3rem]). This could potentially be happening in other places as well, not just in the RSS feed. Is it possible to prevent these plugin shortcodes from appearing as-is in the RSS feed (and possibly elsewhere) during its generation? Thanks!

**Comment by dziudek** [comment]
@followmyrandomness - I will check why these shortcodes are not parsed during generating RSS and most probably we will add support for rendering them, because it can lead to more unexpected side effects if somebody rely on such features

**Comment by bobmitro** [comment]
Fixed in Feather Icons [v.1.4.0](https://marketplace.getpublii.com/plugins/feather-icons/) and Lucide Icons [v.1.1.0](https://marketplace.getpublii.com/plugins/lucide-icons/)
